steps/container.py

```diff
@@ -35,7 +35,8 @@
 # A future version of Cekit will expose this to us, for now we hard-code
 DOCKER_API_VERSION = "1.35"
 
-d = docker.APIClient(version=DOCKER_API_VERSION)
+base_url = os.environ.get("DOCKER_HOST", 'unix:///var/run/docker.sock')
```

Comment:
I was wondering whether that should use `docker.constants.DEFAULT_UNIT_SOCKET` but that value is `http+unix:///var/run/docker.sock`.

Reply:
Well, that value works fine (for Docker), so why not? Pushed a commit to use it.